developer-docs-site/docs/move/book/packages.md

```diff
@@ -290,3 +290,81 @@ a_move_package
 
 See the `move-package` crate for more information on these data structures and
 how to use the Move package system as a Rust library.
+
+## Using Bytecode for Dependencies
+
+Move bytecode can be used as a dependencies when the Move source code for those dependencies are not available locally. To use this feature, you will need co-locate the files in directories at the same level and then specify their paths in the corresponding `Move.toml` files.
+
+## Requirements and limitations
+
+Using local bytecode as dependencies requires bytecode files to be downloaded locally, and the actual address for each named address must be specified in either `Move.toml` or through `--named-addresses`.
+
+Note, both `aptos move prove` and `aptos move test` commands, currently, do not support bytecode as dependencies.
+
+## Recommended structure
+
+We use an example to illustrate the dev flow of using this feature. Suppose we want to compile the package `A` . The package layout is:
+```rust
+./A
+├── Move.toml
+├── sources
+  ├ A.move
+```
+
+`A.move` is defined below, depending on the modules `Bar`and `Foo`:
+
+```rust
+module A::A {
```

Comment:
As a newcomer I found this `A::A` very confusing. What if we did this instead to make it clear which is an address placeholder and which is a module.
```suggestion
module AddressA::ModuleA {
```